src/publicapi.js

```diff
@@ -158,6 +158,10 @@ function getInterface(v) {
       if (this.__controller.blurred) this.__controller.cursor.hide().parent.blur();
       return this;
     };
+    _.empty = _.clear = function() {
+      this.select().__controller.backspace();
+      return this;
+    };
```

Comment:
Can we not have `.clear()` as an alias? API methods are a bit more of a liability than LaTeX commands.

Also, I'd expect this to be an inefficient way to empty the root (`.select()` in particular is relatively expensive since it wraps things in a `<span class="selection">` and writes the LaTeX into a textarea and selects it (since that's how cut and copy work)), I'd recommend something like what `renderLatexMath` does if passed an empty LaTeX, which is basically 5 lines, https://github.com/mathquill/mathquill/blob/master/src/services/latex.js#L108-L109 and https://github.com/mathquill/mathquill/blob/master/src/services/latex.js#L124-L128